src/Renode/Network/NetworkServer/Modules/IcmpServerModule.cs

```diff
@@ -71,8 +71,12 @@ public void HandleIcmpPacket(Action<EthernetFrame> FrameReady, IPv4Packet packet
                 ((IPv4Packet) packet.ParentPacket).SourceAddress);
 
             // We create the ICMP response packet that will be sent in the IPv4 packet
+            // and we asing ID, Data and, Sequence the same like in the replay packet
             var icmpPacketResponse =
```

Comment:
Export each stage of the packet creation to a function so that the code remains readable
[Uncle Bob](https://dev.to/shoiabakbar/writing-functions-uncle-bob-s-principal-3nd9)